velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -716,7 +716,9 @@ class PlanBuilder {
   /// "row_number() over (order by b) as a"
   /// "row_number() over (partition by a order by b
   ///  rows between a + 10 preceding and 10 following)"
-  PlanBuilder& window(const std::vector<std::string>& windowFunctions);
+  PlanBuilder& window(
+      const std::vector<std::string>& windowFunctions,
+      bool isStreamingWindow = false);
```

Comment:
I wonder if it would be cleaner to keep this method as is, but add another one : streamingWindow(windowFunctions).

Reply:
Updated.